src/lib.rs

```diff
@@ -50,14 +50,22 @@
 //!
```

Comment:
Ah we ran into this issue with rustfmt messing up the markdown table in the bson repo too. We'll need to add [`ignore = ["src/lib.rs"]`](https://github.com/mongodb/bson-rust/blob/master/rustfmt.toml#L10) to our `rustfmt.toml` to prevent this. As part of that, can you revert the formatting changes to this file?